README.md

```diff
@@ -4,12 +4,12 @@ A command-line tool to score and rank contributors
 based on their activity over a set of projects
 by analyzing Git histories.
 
+## Usage
```

Comment:
I think this part is too short to be a section. And calling it `Usage` looks redundant with the `Basic Usage` section.